pkgs/applications/graphics/sane/backends/default.nix

```diff
@@ -1,10 +1,102 @@
-{ callPackage, fetchurl, ... } @ args:
+{ stdenv, lib, fetchurl
+, gettext, pkgconfig, python3
+, avahi, libgphoto2, libieee1284, libjpeg, libpng, libtiff, libusb1, libv4l, net-snmp
+, curl, systemd, libxml2, poppler
 
-callPackage ./generic.nix (args // rec {
-  version = "1.0.30";
+# List of { src name backend } attibute sets - see installFirmware below:
+, extraFirmware ? []
+
+# For backwards compatibility with older setups; use extraFirmware instead:
+, gt68xxFirmware ? null, snapscanFirmware ? null
```

Comment:
Should we add an assert to remove those?

Reply:
I'd rather not touch this stuff in this PR.